Makefile

```diff
@@ -27,9 +28,20 @@ testinputs: util
 	${UTIL} -make-dc -cert-in ${TESTDATA_DIR}/example.crt -key-in ${TESTDATA_DIR}/example.key -out ${TESTDATA_DIR}/dc.txt
 	${UTIL} -make-ech -out ${TESTDATA_DIR}/ech_configs -key-out ${TESTDATA_DIR}/ech_key -host client-facing.com
 
+dcvectors: util
+	mkdir -p ${DCVECTORS_DIR}
+	${UTIL} -make-root -out ${DCVECTORS_DIR}/root.crt -key-out ${DCVECTORS_DIR}/root.key -host root.com
+	${UTIL} -make-intermediate -cert-in ${DCVECTORS_DIR}/root.crt -key-in ${DCVECTORS_DIR}/root.key -out ${DCVECTORS_DIR}/example.crt -key-out ${DCVECTORS_DIR}/example.key -host example.com
+	${UTIL} -make-intermediate -cert-in ${DCVECTORS_DIR}/root.crt -key-in ${DCVECTORS_DIR}/root.key -out ${DCVECTORS_DIR}/client_facing.crt -key-out ${DCVECTORS_DIR}/client_facing.key -host client-facing.com
+	${UTIL} -make-dcvectors -cert-in ${DCVECTORS_DIR}/example.crt -key-in ${DCVECTORS_DIR}/example.key -dc-algo 0x0807 -out ${DCVECTORS_DIR}/dc_ed25519.txt
+	${UTIL} -make-dcvectors -cert-in ${DCVECTORS_DIR}/example.crt -key-in ${DCVECTORS_DIR}/example.key -dc-algo 0x0403 -out ${DCVECTORS_DIR}/dc_ecdsa_p256.txt
+	${UTIL} -make-dcvectors -cert-in ${DCVECTORS_DIR}/example.crt -key-in ${DCVECTORS_DIR}/example.key -dc-algo 0x0503 -out ${DCVECTORS_DIR}/dc_ecdsa_p384.txt
+	${UTIL} -make-dcvectors -cert-in ${DCVECTORS_DIR}/example.crt -key-in ${DCVECTORS_DIR}/example.key -dc-algo 0x0603 -out ${DCVECTORS_DIR}/dc_ecdsa_p512.txt
```

Comment:
Is `-make-dcvectors` meant to replace `-make-dc`? It seems like it can, since the new option is more general.

Reply:
I'll unify them ;)